samples/system-test/similarProducts.v1p3beta1.test.js

```diff
@@ -21,12 +21,14 @@ const productSearch = new vision.ProductSearchClient();
 const assert = require('assert');
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 const cmd = `node similarProducts.v1p3beta1.js`;
-const cwd = path.join(__dirname, `..`);
+const cwd = path.join(__dirname, `..`, `productSearch`);
 const filter = ['', 'style=womens'];
-const localPath = './resources/shoes_1.jpg';
-const gcsUri = 'gs://nodejs-docs-samples/product-search/shoes_1.jpg';
+const localPath = './../resources/shoes_1.jpg';
+const gcsUri = 'gs://product-search-node/shoes_1.jpg';
```

Comment:
Same question as above - what project does this bucket live in?

Reply:
Fixed